spec/order_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'order'
+
+describe Order do
+
+  let(:new_dish) { {name: "pizza", quantity: 2, price: 14, accumulated_prize: 28} }
+  let(:new_dish2) { {name: "not_included_dish", quantity: 1, price: 14, accumulated_price: 14} }
+
+  describe '#initialize' do
+  	it 'is expected to start with an empty basket' do
+  	  expect(subject.basket).to be_empty
+  	end
+  end
+
+  describe '#add' do
+  	it {is_expected.to respond_to(:add).with(2).argument}
+
+  	it 'adds the dish to the basket' do
+  	  subject.add("pizza", 2)
+  	  expect(subject.basket).to include new_dish
+  	end
+
+  	it 'is not included in the menu, so it is not added the dish to the basket' do
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)